src/views/new-evaluation/newEvaluation.js

```diff
@@ -213,90 +216,298 @@ const Footer = function(option = {}){
+                                            labelTitle:"Access modifiers"
+                                        },
+                                        {
+                                            selectName:"Design Patterns",
+                                            labelTitle:"Design Patterns"
+                                        },
+                                        {
+                                            selectName:"Issue Tracking",
+                                            labelTitle:"Issue Tracking"
+                                        },
+                                        {
+                                            selectName:"Polymorphism",
+                                            labelTitle:"Polymorphism"
+                                        },
+                                        {
+                                            selectName:"RegEx",
+                                            labelTitle:"RegEx"
+                                        }],
+                            Title:"OOP, Design Patterns",
+                            boxID: "OOP"
                         },
                         {
-                            sContent: selectContent,
-                            selectName :["Protocol","Response codes","REST","Headers","Request methods", "Issue Tracking",
-                                        ],
-                            labelTitle :["Protocol","Response codes","REST","Headers","Request methods", "Sessions &#38; Cookies",
-                                        ]
+                            selectContent: __selectContent,
+                            selectBox:[{
+                                            selectName:"Protocol",
+                                            labelTitle:"Protocol"
+                                        },
+                                        {
+                                            selectName:"Response codes",
+                                            labelTitle:"Response codes"
+                                        },
+                                        {
+                                            selectName:"REST",
+                                            labelTitle:"REST"
+                                        },
+                                        {
+                                            selectName:"Headers",
+                                            labelTitle:"Headers"
+                                        },
+                                        {
+                                            selectName:"Request methods",
+                                            labelTitle:"Request methods"
+                                        },
+                                        {
+                                            selectName:"Issue Tracking",
+                                            labelTitle:"Sessions &#38; Cookies"
```

Comment:
aaaaa..... I really don't know , I mostly try to make it auto-mode when I complete this "manually", but yes seems odd to be Issues Tracking and sesions O.o my  bad